packages/skill-lesson/trpc/routers/invoices.ts

```diff
@@ -20,6 +21,12 @@ export const invoicesRouter = router({
 
       try {
         if (merchantCharge && merchantCharge.identifier) {
+          const {stripe} =
+            merchantCharge.merchantAccount.label === 'egghead-stripe' &&
+            eggheadReadOnlyContext
+              ? eggheadReadOnlyContext
+              : defaultStripeContext
```

Comment:
logntime issue that prevented legacy customers from accessing old invoices that I fixed while in here